cardano-wallet.cabal

```diff
@@ -83,9 +83,8 @@ executable cardano-wallet-server
       NoImplicitPrelude
       OverloadedStrings
   ghc-options:
-      -threaded -rtsopts
```

Comment:
We gotta need this :sweat_smile:  ... If the wallet server isn't threaded, it isn't going to do much serving requests.

Reply:
Oops 😅 was supposed to be temporary